doc/classes/AnimationPlayer.xml

```diff
@@ -28,7 +29,7 @@
 			<argument index="0" name="delta" type="float">
 			</argument>
 			<description>
-				Shifts position in the animation timeline. Delta is the time in seconds to shift. Events between the current frame and [code]delta[/code] are handled.
+				Shifts position in the animation timeline and immediately updates the animation. Delta is the time in seconds to shift. Events between the current frame and [code]delta[/code] are handled.
```

Comment:
We can use `[code]` for both occurrences of the `delta` parameter while we're at it :slightly_smiling_face:

```suggestion
				Shifts position in the animation timeline and immediately updates the animation. [code]delta[/code] is the time in seconds to shift. Events between the current frame and [code]delta[/code] are handled.
```

Starting a sentence with a lowercase character is fine here, as the first word is an identifier.

Reply:
Committed in 550a3ff.